### PR TITLE
Update README to match the new code

### PR DESCRIPTION
**Pull request by saschanaz**
This should match better with the code after #1034.

### CODE REVIEW

File: README.md
Comment (HolgerJeromin) [inline README.md:47]:
```suggestion
It's possible that the automated algorithm decided that it's not well supported by browsers and thus removed it. Say we we want to add a new interface named `Foo`. Check if there is a document about that interface in [MDN](https://developer.mozilla.org/). If there is, see the browser compatibility section and check whether it's supported by two or more browsers. (Chrome, Firefox, and Safari are the relevant browsers. Support from multiple Chromium-based browsers are counted as one support.)
```

Comment (orta) [reply README.md:47]: Maybe "Support from multiple Chromium-based browsers are counted as one browser" - using support at the end  doesn't really fit in English grammar.

**Comment by saschanaz** [comment]
@orta @sandersn Do you see something to add or modify here? 👀

File: README.md
Comment (sandersn) [inline README.md:66]: This paragraph is mostly still true, right? I know we are less conservative than before, but the idea is that every update to `@types/web` will *eventually* ship with Typescript by default means that there are some changes that will not be OK until there's a way to publish other packages from here.

**Review by sandersn — CHANGES_REQUESTED** [review]
I'm not too sure about the removal of heuristics, which still somewhat apply. @orta can you take a look too?

File: README.md
Comment (saschanaz) [reply README.md:70]: It can't be "more up to date" anymore since the data itself is being constantly autoupdated while the JSON file is handled manually. Removing that word makes it "have better definitions" which is a vague expression.

File: README.md
Comment (saschanaz) [reply README.md:66]: Yes, maybe it's good to keep the sentence for maintainers. It has minimal meaning for contributors though, since nearly all changes will now be from webref and MDN rather than manual PRs.

Comment (saschanaz) [reply README.md:66]: >I know we are less conservative than before

I'm not sure, as quite a lot types are removed for lack of broad browser support?

**Review by orta — APPROVED** [review]
OK, yeah, I think this probably what we want

File: README.md
Comment (saschanaz) [reply README.md:70]:
```suggestion
- `overridingTypes.jsonc`: types that are defined in the spec but have TypeScript-friendly modifications in the json files.
```

**Comment by saschanaz** [comment]
Cool, I'll merge this if @sandersn also agrees.

**Review by sandersn — APPROVED** [review]

**Comment by saschanaz** [comment]
LGTM

**Pull request merged**
